### ISSUE TITLE
Write examples section into matthew.html - Matthew

### ISSUE THREAD

**Issue by mariannegoldin** [josephwanderer/itc134fl17-waffle-pie#10]

**Comment by BuddBikes** [comment]
As of now the examples section of Eve's white paper consists exclusively of the png. image file listing industries currently applying machine learning and a brief paragraph describing the utilization of object recognition capabilities for autonomous vehicles.

I had proposed adding further information that I had gathered in my research for my white paper on additive manufacturing.  I think this will be necessary to flesh out the examples section to effectively write a PIE formatted set of paragraphs to hte report

**Comment by mariannegoldin** [comment]
Agreed. I think it is a good idea, as you propose, in the Examples section to add in your NN/Manufacturing content in addition to a summary of the existing ML paper content (image and text).

**Comment by mariannegoldin** [comment]
To confirm, Matthew will be writing the Examples section with the added Manufacturing content added in.

**Comment by BuddBikes** [comment]
Good morning,
I have a pull request pending for a rough draft of my examples section. I
have to be off the computer till class at 1 in  room 1130 today.
Let me know what you think of what I have so far and any advice for tweaks,
will further editing effect the timeline for getting the page up and
running?  I figure the document can still be live after the assignment is
turned in.  I will continue to work on this but have other classes to
attend all evening and won't likely get to editing
again until tomorrow.

Matt

**Comment by mariannegoldin** [comment]
Hi Matthew, it sounds like you might be making more edits to your section (markup, lists, links, etc), and then we will add the second iteration to the main page.

**Comment by mariannegoldin** [comment]
Would it be possible to add some of the material from the ML Whitepaper Examples section? The image used in the whitepaper provides a good intro of the examples of how ML is used, and the paragraph has some good introductory statements that could be useful.

https://docs.google.com/document/d/1H_GT4azMgxBqCdo1rm95DoWAbEC4gd0KdmIdWz1Z3g8/edit

**Comment by BuddBikes** [comment]
I commited the graphic that the ML White Paper Example section uses....I think it went into the images folder, I'll tie it into my section.  I will get it uploaded to my examples section.  It looks like my section is still isolated from the Waffle Pie Web Page.
Otherwise I'm good with my section going in as is to check the whole page for readability.
